server/issue.go

```diff
@@ -799,6 +801,19 @@ func (s *Server) createPipelineFromIssue(ctx context.Context, issueCreate *api.I
 		return nil, echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("invalid issue type %q", issueCreate.Type))
 	}
 
+	// Return an error if the issue has no task to be executed
```

Comment:
So I guess frontend also does the check and this is the corresponding backend check as another depth of defense (also for CLI, API path later).

If that's the case, I am wondering if we can move this into createPipelineFromIssue directly.

Reply:
Yeah, I think it is part of createPipelineFromIssue().

Reply:
So why we still need this here since we already call createPipelineFromIssue at L337

Reply:
I mean this change is part of createPipelineFromIssue() from L473.

:( github probably doesn't render it well unless we expand the whole file...

Reply:
Oh, I missed. But glad we are on the same camp ^_^